rust-portaudio-sys/src/lib.rs

```diff
@@ -9,6 +9,9 @@
 mod c_library {
     #[link(name = "portaudio")]
     extern {}
+
+    #[link(name = "jack")]
```

Comment:
Can you enable this only for Linux?

https://doc.rust-lang.org/rust-by-example/attribute/cfg.html